src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -32,6 +33,14 @@ export const enabledErc20NetworkTokens: Readable<Erc20Token[]> = derived(
 		$enabledNetworkTokens.filter(({ standard }) => standard === 'erc20') as Erc20Token[]
 );
 
+export const enabledIcNetworkTokens: Readable<IcToken[]> = derived(
+	[enabledNetworkTokens],
+	([$enabledNetworkTokens]) =>
+		$enabledNetworkTokens.filter(
+			({ standard }) => standard === 'icp' || standard === 'icrc'
```

Comment:
here i prefer this instead of

`['icp', 'icrc'].includes(....`

because this is type checked, i.e. it would give an error if there was a mistype in the right-hand side of the comparison that does not match one of the possible values of `TokenStandard`.

Reply:
If we ends up using this derived store, maybe provide it in a separate PR?

Reply:
Done:

- PR https://github.com/dfinity/oisy-wallet/pull/2213